feedthefox/users/adapters.py

```diff
@@ -36,8 +36,18 @@ def populate_user(self, request, sociallogin, data):
         user = super(FeedTheFoxSocialAdapter, self).populate_user(request,
                                                                   sociallogin,
                                                                   data)
+        query_email = user.email
+
+        if sociallogin.account.provider == 'github':
+            email_addresses = sociallogin.account.extra_data.get('email_addresses')
+            verified_address = [address['email'] for address in email_addresses
```

Comment:
Is there any way that user might have more than one primary addresses? If not instead of using list I suggest we get directly the string.